common/lib/common-utils/src/logger.ts

```diff
@@ -41,3 +43,31 @@ export class TelemetryNullLogger implements ITelemetryLogger {
     public sendPerformanceEvent(event: ITelemetryPerformanceEvent, error?: any): void {
     }
 }
+
+/**
+ * Walk an object's enumerable properties to find those fit for telemetry.
+ */
+export function getValidTelemetryProps(obj: any): ITaggableTelemetryProperties {
```

Comment:
IMHO, this does not belong here (in this form). It should be a private method of LoggingError, and not using any (obj arg usage should be replaced with "this").

Reply:
Yeah I agree this shouldn't be exported, @ksbrar you can remove the changes to this file then.

Reply:
Removed.